avalanche/deployments/outputs/43114-production.json

```diff
@@ -0,0 +1,129 @@
+      "constructorArgs": [],
+      "libraries": {}
+    },
+    "1": {
+      "id": "0x2e506effce8b4a8b69bd7c35fbce3bbdb8ec2a1fc8bc5fbd9e066e0142b7d7bf",
+      "name": "DelegatedManagerFactory",
+      "timestamp": 1652890661664,
+      "verified": true,
+      "description": "Deployed DelegatedManagerFactory",
+      "contractAddress": "0xaF31F117439bDEfa52B4e2fa47C0B800a6D10c70",
+      "constructorArgs": [
+        "0x659641F71525b02bFe7203F3e5113c48E6736540",
+        "0xC59E05a299b2FBD418DA8E67CB51DA05c67bDB79",
+        "0xbaae6346DE5915886B1759c7e68272352b524161"
+      ],
+      "libraries": {}
+    },
+    "2": {
+      "id": "0x03334c96df387f7352e6c0030a100a25172e63781a7993c2ef9cd3dbda9e0595",
+      "name": "IssuanceExtension",
+      "timestamp": 1652890668135,
+      "verified": true,
+      "description": "Deployed IssuanceExtension",
+      "contractAddress": "0xea1bD0ae3Ec45e32a6090eDb445dACFc083A8db6",
+      "constructorArgs": [
+        "0x659641F71525b02bFe7203F3e5113c48E6736540",
+        "0x6808f8D7DF3295BdA11998dd675C1fd2CedCbF5f"
+      ],
+      "libraries": {}
+    },
+    "3": {
+      "id": "0x42f87f38d263c0f3a54b9b822cfcc3d5c96cb1e47045e8e037f87d257361bd06",
+      "name": "StreamingFeeSplitExtension",
+      "timestamp": 1652890674478,
+      "verified": true,
+      "description": "Deployed StreamingFeeSplitExtension",
+      "contractAddress": "0x5018AC1223CebdE29bE41b07b858F3846390B052",
+      "constructorArgs": [
+        "0x659641F71525b02bFe7203F3e5113c48E6736540",
+        "0x0c9184c1190b73dFFCEea44aE196E11CD2eC4a6C"
+      ],
+      "libraries": {}
+    },
+    "4": {
+      "id": "0x0a86ae33a2d07ee6748d9d4b12020f62789ac2c8cc63f5f1ba259b90b409483a",
+      "name": "TradeExtension",
+      "timestamp": 1652890681380,
+      "verified": true,
+      "description": "Deployed TradeExtension",
+      "contractAddress": "0xd33042aa2d9aEd46d741F2ff063de6516DBbFdE5",
+      "constructorArgs": [
+        "0x659641F71525b02bFe7203F3e5113c48E6736540",
+        "0xD9ABfa88beacC41FA1119210e39fCd49b58D5ef8"
+      ],
+      "libraries": {}
+    },
+    "5": {
+      "id": "0x30d702d8d8bca5cb33222e5fbc4217f2d79c506522c64042c867f3c624740851",
+      "timestamp": 1652890692607,
+      "description": "Initialized ManagerCore with DelegatedManagerFactory, IssuanceExtension, StreamingFeeSplitExtension, and TradeExtension"
+    },
+    "6": {
+      "id": "0x226f6f9af6d97b33ec6c7d4d6ca1b0bf59eaae66f0ccc7dba1aaa82e7b54ac83",
+      "name": "BatchTradeExtension",
+      "timestamp": 1652890698919,
+      "verified": true,
+      "description": "Deployed BatchTradeExtension",
+      "contractAddress": "0xa415bcFD9d8B70DaFEe7d53cDDB636e1395683C7",
+      "constructorArgs": [
+        "0x659641F71525b02bFe7203F3e5113c48E6736540",
+        "0xD9ABfa88beacC41FA1119210e39fCd49b58D5ef8",
+        [
+          "ZeroExApiAdapterV5"
+        ]
+      ],
+      "libraries": {}
+    },
+    "7": {
```

Comment:
blundered here by not including a skip production check. will require a multi-sig transaction

Reply:
Ah ok!